addons/point_of_sale/models/pos_order.py

```diff
@@ -334,7 +334,7 @@ def _default_pricelist(self):
     date_order = fields.Datetime(string='Order Date', readonly=True, index=True, default=fields.Datetime.now)
     user_id = fields.Many2one('res.users', string='Salesman', help="Person who uses the cash register. It can be a reliever, a student or an interim employee.", default=lambda self: self.env.uid)
     amount_tax = fields.Float(compute='_compute_amount_all', string='Taxes', digits=0)
-    amount_total = fields.Float(compute='_compute_amount_all', string='Total', digits=0)
+    amount_total = fields.Float(compute='_compute_amount_all', string='Total', digits=0, store=True)
```

Comment:
@mart-e @kebeclibre I'd like to use amount_total in a read_group method, for reporting purposes, so I need to store this field. (I'm not sure I understand why this isn't already in a report)

The compute method doesn't seem to have all the triggers set correctly, and it already depends on computed fields (I'm not sure it works), to set all triggers correctly it'll probably need like 15 dependencies.

Do you know why all these amounts are computed and not stored by default and computed once ?
Are we supposed to be able to change the Total/Paid/Taxed/Returned amount ? It doesn't seem normal to me.
(Also, why is amount_paid not readonly in draft mode ?)